config.ts

```diff
@@ -0,0 +1,8 @@
+export interface Config {
```

Comment:
Should we make the config as similar to Cloud Trace config as possible? See https://github.com/GoogleCloudPlatform/cloud-trace-nodejs. For example, that seems to use an environment variable to specify the project ID, and has the service parameter in a nested structure.